pyvista/plotting/charts.py

```diff
@@ -4458,3 +4458,7 @@ def __iter__(self):
         """Return an iterable of charts."""
         for chart in self._charts:
             yield chart
+
+    def __del__(self):
```

Comment:
```suggestion
    def __del__(self):  # pragma: no cover
```

Reply:
This actually led me down a rabbit hole that I'll open a PR about soon, but I'm still investigating (this is why I haven't yet looked at the other PRs as planned, sorry). I should've pushed this for the time being, thanks for adding it.